implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogConfig.java

```diff
@@ -56,12 +58,56 @@ default String hostTag() {
         return getString(this, "hostTag").orElse("instance");
     }
 
+    /**
+     * Truthy if the configuration is defined for a dogstatsd reporting configuration.
+     * @return Returns true if this is a non https scheme target, to indicate to talk via
+     * dogstatsd.
+     */
+    default boolean isStatsd() {
```

Comment:
I'm not sure when a user would override the value for this. Is this method just for our convenience? If so, it should probably be a private method in `DatadogMeterRegistry` instead so it is not exposed to users.